Twig/Extension/TextFormatterExtension.php

```diff
@@ -20,6 +20,8 @@ class TextFormatterExtension extends \Twig_Extension
 {
     protected $pool;
 
+    protected $environment;
```

Comment:
Please add a php doc block for this.

Yes I know. No php doc on another property. That is not a reason! :-)